pkg/oc/cli/cmd/login/login.go

```diff
@@ -152,6 +153,8 @@ func (o *LoginOptions) Complete(f *osclientcmd.Factory, cmd *cobra.Command, args
 	o.InsecureTLS = kcmdutil.GetFlagBool(cmd, "insecure-skip-tls-verify")
 	o.Token = kcmdutil.GetFlagString(cmd, "token")
 
+	o.Silent = kcmdutil.GetFlagBool(cmd, "silent")	
```

Comment:
Seems unnecessary, @juanvallejo why is this filling out fields when they should be filled via the pointer stuff?